Source/Coop/Components/CoopGameComponents/CoopGameComponent.cs

```diff
@@ -598,6 +598,26 @@ void ProcessQuitting()
                 return;
             }
         }
+        
+        /// <summary>
+        /// Handles a player going MissingInAction by staying in the game too long.
+        /// </summary>
+        void ProcessMissingInAction()
+        {
+            var gameTime = Singleton<AbstractGame>.Instance?.GameTimer.EscapeDateTime;
+            if (gameTime != null)
+            {
+                var remainingTime = (float)(gameTime.Value - GClass1293.UtcNow).TotalSeconds;
+                if (remainingTime <= 0.5f)
+                {
+                    Singleton<ISITGame>.Instance.Stop(
+                        Singleton<GameWorld>.Instance.MainPlayer.ProfileId
+                        , ExitStatus.MissingInAction
```

Comment:
You should check if the player is dead or extracted first before give him a "MiA", or someone who already died or extracted will having incorrect exit status if they were AFK (specifically Host).
```python
EFT.Player player = Singleton<GameWorld>.Instance.MainPlayer;
bool isAlive = player.HealthController.IsAlive;
bool isExtracted = coopGame.ExtractedPlayers.Contains(player.ProfileId);
```